webportal/src/app/user/user-register/user-register.component.ejs

```diff
@@ -49,19 +49,19 @@
                 <label>
                   <input type="checkbox" name="admin">Admin user</label>
               </div>
-              <button class="btn btn-lg btn-primary btn-block">Update account</button>
+              <button class="btn btn-lg btn-primary btn-block">Change password</button>
             </form>
           </div>
           <div class="tab-pane" id="update_virtual_cluster">
             <form id="form-update-virtual-cluster" class="form-register">
-              <h2 class="form-register-heading">Change Password</h2>
+              <h2 class="form-register-heading">Update VirtualClusters</h2>
               <label for="inputUsername" class="sr-only">Username</label>
               <input type="username" name="username" id="update-virtual-cluster-input-username" class="form-control" placeholder="Username"
                 required autofocus>
               <label for="inputVirtualCluster" class="sr-only">VirtualCluster</label>
               <input type="virtualCluster" name="virtualCluster" id="update-virtual-cluster-input-virtualCluster" class="form-control"
                 placeholder="VirtualClusters (e.g. vc1,vc2)">
-              <button class="btn btn-lg btn-primary btn-block">Change Password</button>
+              <button class="btn btn-lg btn-primary btn-block">Update virtualClusters</button>
```

Comment:
>v [](start = 70, length = 1)

Uppercase "V" should be.

Reply:
I referred to github webportal, it just uppercase first letter of first word like "Update profile" "Add single comment"